dlt/pipeline/pipeline.py

```diff
@@ -903,10 +903,12 @@ def _get_destination_client_initial_config(self, destination: DestinationReferen
         return client_spec(credentials=credentials)
 
     def _get_destination_clients(self,
-        schema: Schema,
+        schema: Schema = None,
         initial_config: DestinationClientConfiguration = None,
         initial_staging_config: DestinationClientConfiguration = None
     ) -> Tuple[JobClientBase, JobClientBase]:
+        if not schema:
```

Comment:
why you allow for that?

Reply:
It made a few locations in the code easier to write, I can also not allow this and put the default schema everywhere manually where I need it. It's up to you.

Reply:
I'd require schema here. this methods is already doing too much inside

Reply:
I made it mandatory again